Vulkan-Projects.adoc

```diff
@@ -0,0 +1,162 @@
+// Copyright 2017-2021 The Khronos Group Inc.
+// SPDX-License-Identifier: CC-BY-4.0
+
+# Map of Vulkan project GitHub repositories
+
+Vulkan is a big place and there's a lot out there that is useful! In no particular order...
+
+## Khronos/Vulkan® Specification
+Vulkan® Specification - formal documentation of the Vulkan API. This includes the main API Specification, the reference (man) pages, the XML API Registry, and related tools and scripts.
+
+* Landing page: https://www.vulkan.org/
+* Website: https://www.khronos.org/registry/vulkan/
+* Repository: https://github.com/KhronosGroup/Vulkan-Docs
+* License: [mixed](https://github.com/KhronosGroup/Vulkan-Docs/blob/1.0/COPYING.md) &mdash; approximately:
+    * Proprietary Khronos license for the Vulkan Specification published by Khronos on its website
+    * CC-BY 4.0 for asciidoctor sources for the Specification and related documents such as the reference pages, style guide, and XML schema documentation.
+    * Apache 2 for code, including the XML registry defining API interfaces
+* Contacts:
+    * [Vulkan forum](https://forums.khronos.org/forumdisplay.php/114-Vulkan)
+* Compiler requirements: python 3 (3.4.2 known working), ruby 2 (2.3.3 known working)
```

Comment:
I'm going to go ahead and merge this, but I think better to point this to the khronosgroup/docker-images:asciidoctor-spec
 Docker image on dockerhub, because the toolchain is so much more complex than just the languages used. People can roll their own environment for building the spec (I do), but using the image is a much quicker way to do it.

Reply:
done